ow_faults_detection/src/FaultDetector.cpp

```diff
@@ -0,0 +1,137 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#include "ow_faults_detection/FaultDetector.h"
+#include <algorithm>
+
+using namespace std;
+using namespace ow_lander;
+
+constexpr std::bitset<10> FaultDetector::isCamExecutionError;
+constexpr std::bitset<10> FaultDetector::isPanTiltExecutionError;
+constexpr std::bitset<10> FaultDetector::isArmExecutionError;
+constexpr std::bitset<10> FaultDetector::isPowerSystemFault;
+
+constexpr std::bitset<3> FaultDetector::islowVoltageError;
+constexpr std::bitset<3> FaultDetector::isCapLossError;
+constexpr std::bitset<3> FaultDetector::isThermalError;
```

Comment:
The "std::" prefix is not needed, because of the `using namespace std` (which earlier I suggested replacing with more specific clauses, like `using std::bitset`

Reply:
i left the using namespace std since that was the default in fault injector. I can update everything to using if that appears better

Reply:
Something should change here.  I suggest the safer and clearer approach:

- get rid of `using namespace std`
- add `using std::bitset`
- get rid of the `std::` prefix here
- add more `using std::foo` statements for foo's that are used repeatedly, else add `std::` to those foo's.
-

Reply:
I think these were updated yesterday to removing namespace std. I'll double check the remaining std that might be lingering.